nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -115,6 +115,21 @@ in
         description = "True if ZFS filesystem support is enabled";
       };
 
+      unsafeDisableVersionCheck = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          This will disable the kernel version compatibility check of the ZFS
+          kernel module that is normally in place. Hence this allows the user to
+          compile (or at least attempt to compile) the ZFS kernel module against
+          kernel versions that are not officially supported by upstream.
+
+          <emphasis>Caution</emphasis>: You are entering UNSUPPORTED
+          territory. Here be dragons, pool corruption and data loss! You have
+          been warned.
```

Comment:
Highly doubt anyway will read this...

Reply:
Okay, should I remove it?

Reply:
I read the auto-generated option manpages *all the time*.

Reply:
I guess it's better to have it, than not have it